test/environment_test.rb

```diff
@@ -461,7 +461,7 @@
     end
   end
 
-  describe '#code_reloading?' do
+  describe '#code_reloading?', engine: :mri do
```

Comment:
I understand this was needed because of limitations of the JRuby environment, but loosing code reloading seem like a big shot into JRuby as a development target. I believe it's a Good Thing™ to open an issue to further down some investigation about alternatives or at least improve the documentation to warn future users about this shortcoming.

Reply:
@joneslee85 `entr` support is removed from hanami way before 1.0. Is there alternative for jRuby?